cmd/protecodeExecuteScan.go

```diff
@@ -336,5 +336,10 @@ func getTarName(config *protecodeExecuteScanOptions) string {
 	}
 	// replace unwanted chars
 	fileName = strings.ReplaceAll(fileName, "/", "_")
+	sha256 := "@sha256"
+	if strings.Contains(fileName, sha256) {
+		fileName = strings.ReplaceAll(fileName, ":", "_")
+		fileName = strings.ReplaceAll(fileName, sha256, "")
```

Comment:
Maybe it would be better to use Replace instead of ReplaceAll. If somebody has more than one occurrence of `@sha256` then it should probably fail rather sooner than later...

Reply:
done as requested